src/context/AppContext.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext } from "react";
+import defaultValue from "./defaultValue.tsx";
+
+const AppContext = createContext(defaultValue);
+
+const AppContextProvider = ({
+  children,
+}: {
+  children: JSX.Element;
+}): JSX.Element => {
+  const languages = ["en-US", "es-AR"];
+  const [state, setState] = React.useState({
+    languages: languages,
+    locale: navigator.language.startsWith("es", 0) ? "es-AR" : "en-US",
```

Comment:
instead get locale directly here, maybe we could do a util method, something like ```getLanguage```